react/features/virtual-background/functions.js

```diff
@@ -50,30 +51,33 @@ export const toDataURL = async (url: string) => {
  * Resize image and adjust original aspect ratio.
  *
  * @param {Object} base64image - Base64 image extraction.
+ * @param {Function} callback - Function that will return the resized image.
  * @param {number} width - Value for resizing the image width.
  * @param {number} height - Value for resizing the image height.
  * @returns {Object} Returns the canvas output.
  *
  */
-export async function resizeImage(base64image: any, width: number = 1920, height: number = 1080) {
+export function resizeImage(base64image: any, callback: Function, width: number = 1920, height: number = 1080) {
```

Comment:
Please make this async and return a Promise instead, it better matches the flow of the code that uses this.